src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -605,6 +602,12 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
   <data name="HttpsConnectionEstablished" xml:space="preserve">
     <value>Connection "{connectionId}" established using the following protocol: {protocol}</value>
   </data>
+  <data name="HTTP2DefaultCiphersInsufficient" xml:space="preserve">
```

Comment:
nit: All other keys are `Http2Blah`, not `HTTP2Blah`. New keys should follow the existing convention.

Reply:
I based it off of HTTP2NoTlsOsx which had that particular casing. I don't think we are terribly consistent with namings but I'll change this file at least.